spec/helpers/rpc_mock/client.lua

```diff
@@ -0,0 +1,68 @@
+-- by importing helpers, we ensure the kong PDK module is initialized
+local helpers = require "spec.helpers"
+local rpc_mgr = require("kong.clustering.rpc.manager")
+local default_cert = require("spec.helpers.rpc_mock.default").default_cert
+local uuid = require "kong.tools.uuid"
+
+
+local _M = {}
+
+
+local default_dp_conf = {
+  role = "data_plane",
+  cluster_control_plane = "localhost:8005",
+}
+
+setmetatable(default_dp_conf, { __index = default_cert })
+local default_meta = { __index = default_dp_conf, }
+
+
+local function do_nothing() end
+
+
+local function client_stop(rpc_mgr)
+  -- a hacky way to stop rpc_mgr from reconnecting
+  rpc_mgr.try_connect = do_nothing
+
+  -- this will stop all connections
+  for _, socket in pairs(rpc_mgr.clients) do
+    for conn in pairs(socket) do
+      pcall(conn.stop, conn)
+    end
+  end
+end
+
+
+local function client_is_connected(rpc_mgr)
+  for _, socket in pairs(rpc_mgr.clients) do
```

Comment:
Should we use `table.isempty`?

Reply:
I don't think it's performance critical for a test function

Reply:
`table.isempty` is easy to understand for reviewers.

Reply:
The main concern here is to reduce dependency. Anyway let's do it